src/renderers/dom/shared/ReactDOMDebugTool.js

```diff
@@ -40,9 +41,11 @@ function emitEvent(handlerFunctionName, arg1, arg2, arg3, arg4, arg5) {
 
 var ReactDOMDebugTool = {
   addDevtool(devtool) {
+    ReactDebugTool.addDevtool(devtool);
```

Comment:
I figured we would probably expose both – though maybe it makes more sense to only expose each platform-specific one. Open to other suggestions on how to set ReactDOMUnknownPropertyDevtool up so that we can listen to both kinds of events though.